versions/b-/botan.json

```
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "bc62b88dc0ea58b6121cd0f1d628d1384869af45",
+      "version-string": "2.17.3",
```

```suggestion
      "version": "2.17.3",
```
This also should be `version`.